src/main/java/org/opensearch/flowframework/common/CommonValue.java

```diff
@@ -148,4 +148,20 @@ private CommonValue() {}
     public static final String USER_OUTPUTS_FIELD = "user_outputs";
     /** The template field name for template resources created */
     public static final String RESOURCES_CREATED_FIELD = "resources_created";
+    /** LLM Name for registering an agent */
+    public static final String LLM_FIELD = "llm";
+    /** The tools' field for an agent */
+    public static final String TOOLS_FIELD = "tools";
+    /** The memory field for an agent */
+    public static final String MEMORY_FIELD = "memory";
+    /** The app type field for an agent */
+    public static final String APP_TYPE_FIELD = "app_type";
+    /** The agent id of an agent */
+    public static final String AGENT_ID = "agent_id";
+    /** To include field for an agent response */
+    public static final String INCLUDE_OUTPUT_IN_AGENT_RESPONSE = "include_output_in_agent_response";
+    /** The created time field for an agent */
+    public static final String CREATED_TIME = "created_time";
+    /** The last updated time field for an agent */
+    public static final String LAST_UPDATED_TIME_FIELD = "last_updated_time";
```

Comment:
can you add at least in the constant a reference to agent as we might add this field to our workflows when we finalize the update API. for example: `AGENT_LAST_UPDATED_TIME_FIELD`

Reply:
Since it's a common class, I don't want to keep it specific fields. With other APIs we might need this field again. WDYT?

Reply:
Agreed with keeping this as `LAST_UPDATED_TIME_FIELD`